fetch.js

```diff
@@ -1,6 +1,6 @@
 const https = require("https");
 
-module.exports = function fetch(options, data) {
+module.exports = function fetch(options) {
```

Comment:
The "data" parameter looked unused by all usages of this function. I figured that was for eventual tests, so I removed it here since I did add tests.

Reply:
Hmm, looks like the original author used `data` as a variable/object to hold fetch results output? https://github.com/dev-drprasad/delete-tag-and-release/blob/master/index.js#L76
But perhaps it could have just been named something more commonly seen like `results`? dunno what his ideas were, do you looking through that?

Reply:
Yeah, I think I figured out what happened there. `data` was meant for actually _sending_ data along with the request. I'm going to un-delete that since it's not actually relevant to what I wanted to do: namely add tests.

Reply:
Done.